.github/workflows/cmake.yml

```diff
@@ -0,0 +1,134 @@
+name: CMake
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
```

Comment:
Please change this to `Debug` for push events, and `RelWithDebInfo` for release/manual/pre-merge events (or alternatively don't build on `Debug` at all and only build on `RelWithDebInfo`)